test/js/setup-tests.js

```diff
@@ -1,12 +1,12 @@
-import { configure } from 'enzyme';
-import Adapter from 'enzyme-adapter-react-16';
+import { configure } from 'enzyme'
+import Adapter from 'enzyme-adapter-react-16'
 
-configure({ adapter: new Adapter() });
+configure({ adapter: new Adapter() })
 
 import {
   setGlobalMockFlag,
```

Comment:
I'm not personally a fan of required trailing commas, but don't feel too strongly about it

Reply:
Same here.  I just flipped that linter rule to err on trailing commas.